### DISCUSSION TITLE
Welcome to dotrix Discussions!

### DISCUSSION THREAD

**Discussion by voxelias** [lowenware/dotrix#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by TheTechRobo** [comment]
Can't wait until it's finished, this seems like a great project!

**Reply by voxelias** [reply]
Thank you! It is very inspiring to receive such feedback! Could you please specify what features are the most wanted for you now?

**Reply by TheTechRobo** [reply]
Top on my wish list right now is probably documentation. I really want to start game developing, but can't find much documentation here. 

Thanks for all this work on dotrix. :)

**Reply by voxelias** [reply]
Well, there is a [demo](https://github.com/lowenware/dotrix/blob/main/examples/demo/demo.rs) example, which is documented in code. Could be a good starting point. However, I am working on a Handbook. It will take some time, due to other tasks, but should be released ASAP.

**Reply by TheTechRobo** [reply]
Sounds awesome, can't wait!
